src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -1,56 +1,109 @@
 package seedu.address.ui;
 
+import java.util.LinkedList;
+import java.util.ListIterator;
+
 import javafx.collections.ObservableList;
 import javafx.fxml.FXML;
 import javafx.scene.control.TextField;
+import javafx.scene.input.KeyEvent;
 import javafx.scene.layout.Region;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
- * The UI component that is responsible for receiving user command inputs.
- */
+* The UI component that is responsible for receiving user command inputs.
```

Comment:
Is the change in indentation of the block comment * intentional? ._.

Reply:
prettier, and checkstyle doesn't care.